nano/node/node.cpp

```diff
@@ -1365,26 +1366,33 @@ void nano::block_processor::flush ()
 
 bool nano::block_processor::full ()
 {
+	size_t full_size (node.flags.fast_bootstrap ? 1024 * 1024 : 65536);
```

Comment:
This is a good idea.  Ideally this would be a different PR, but let's leave the set of changes in this one.  We can build something different on top of this.  Maybe fast(1M)/normal(64K)/custom(user-selectable).

Reply:
Yeah, selectable is good idea, will make issue to track this